src/index.html

```diff
@@ -3,11 +3,343 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bang & Olufsen</title>
+    <link
+      rel="shortcut icon"
+      href="./images/favicon.ico"
+      type="image/x-icon"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="home" id="home">
+    <header class="header" id="header">
+      <div class="nav header__nav">
+        <a
+          href="index.html"
+          class="nav__link"
+        >
+          <img
+            class="nav__item logo"
+            src="./images/logo.png"
+            alt="logo"
+          >
+        </a>
+        <div class="nav__links">
+          <a href="tel: +1 234 555-55-55" class="nav__link phone">
```

Comment:
Need to fix format phone everywhere only in href
```suggestion
          <a href="tel:+12345555555" class="nav__link phone">
```